api/requests.js

```diff
@@ -0,0 +1,31 @@
+const countriesUrl = 'http://localhost:3000/countries';
+
+export const getData = async () => {
+  try {
+    const response = await fetch(countriesUrl, {
+      method: 'GET',
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
```

Comment:
Since we use this header for every APi call, let's create some common method, call it for example `request` and pass default parameters there, to not copy-paste them